easybuild/easyconfigs/s/snap/snap_2013-11-29_makefile_correction.patch

```diff
@@ -0,0 +1,65 @@
+drop the hard coded "gcc", also remove the -Werror since there are warining with GCC 
```

Comment:
typo: `warining` should be `warning` (or `warnings` I guess)

(note that fixing this implies a changed SHA256 checksum for the patch file)

Reply:
@boegel fixed the program name, changed it from snap to SNAP.